UKHomeOffice/cerberus-service
# COP-11890 - Number of changes between versions

## Description
This PR extracts the number of version changes between a current version and a previous version and displays this within the accordion header.

## To Test
- Pull & run against dev
- Post a task to dev
- Post an update to the same task excluding (risks & selectors) e.g. for an Accompanied movement, modify the vehicle details and keep track of how many changes made.
- The task should now have an update and in the accordion header, the number of changes should reflect the number changes that was made to the update task.
